services/ens/api.go

```diff
@@ -336,42 +348,134 @@ func (api *API) Price(ctx context.Context, chainID uint64) (string, error) {
 	return fmt.Sprintf("%x", price), nil
 }
 
-func (api *API) Release(ctx context.Context, chainID uint64, txArgs transactions.SendTxArgs, password string, username string) (string, error) {
-	registryAddr, err := api.usernameRegistrarAddr(ctx, chainID)
+// Signs a transaction using appropriate local keystore file that is decrypted with provided password and returns the signature in hex format.
+// The tx that will be signed is the one that was prepared by the last call to
+// PrepareTxForRegisteringEnsUsername or PrepareTxForSettingPublicKey or PrepareTxForReleasingRegisteredEnsUsername.
+func (api *API) SingPreparedTx(password string) (interface{}, error) {
```

Comment:
Why do you return interface{} and not string?

Reply:
While I was developing I used `[]byte`, then `string` and then it remained like that. Np, will change that.